### PR TITLE
Updated NucAminoAligner's generateCmd() method to reflect NucAmino's Interface Upgrade

### PR DESCRIPTION
**Pull request by arodcasanova**
Updated generateCmd() to invoke NucAmino's updated interface. Previously-failing tests are now passing. However, I still need to run the rest of the tests I haven't checked.

### CODE REVIEW

File: Aligner/src/main/java/edu/stanford/hivdb/alignment/NucAminoAligner.java [hunk 1]
```diff
@@ -474,16 +474,15 @@ private static AlignedGeneSeq geneSeqFromReport(
 			executable = "nucamino";
 		}
 		return new String[] {
-			executable,
-			"hiv1b", // hiv1b sub-command
-			"-q", // quiet mode
-			"--gene", "POL", // align input sequence with POL consensus
-			"--output-format", "json",
-			"--indel-codon-opening-bonus", "" + INDEL_CODON_OPENING_BONUS,
```
Comment (philiptzou) [inline Aligner/src/main/java/edu/stanford/hivdb/alignment/NucAminoAligner.java]: Please also remove the unused constants.

Comment (arodcasanova) [reply Aligner/src/main/java/edu/stanford/hivdb/alignment/NucAminoAligner.java]: Updated NucAmino to v0.2.0-beta-2; removed unused constants (thank you for catching that); passed tests via ./gradlew test

Comment (arodcasanova) [reply Aligner/src/main/java/edu/stanford/hivdb/alignment/NucAminoAligner.java]: (NucAmino updated in .travis.yml)

Comment (arodcasanova) [reply Aligner/src/main/java/edu/stanford/hivdb/alignment/NucAminoAligner.java]: spotted error. fixing now.

**Comment by philiptzou** [comment]
All unit tests are passed on Travis. However, can you ensure it locally that `PrettyAlignmentsTest` and `HIVGenotypeResultTest` are also passed? Those two are excluded from TravisCI since they took so long to finish.

**Comment by philiptzou** [comment]
Please create a new branch in your repository and point it to revision b553b360b2d85175f60e85029bb86a8ccfb399b4.

**Pull request closed without merging**